src/main/java/com/kryptnostic/rhizome/core/RhizomeApplicationServer.java

```diff
@@ -21,6 +22,7 @@
     private final        List<Class<?>>                     additionalPods = new ArrayList<>();
     public static final  Class<?>[]                         DEFAULT_PODS   = new Class<?>[] {
             AsyncPod.class,
+            Auth0Pod.class,
```

Comment:
What is this added for?

Reply:
/I think we want to leave it up to extending classes to decide whether to use `Auth0Pod` or not

Reply:
All of them are registering Auth0Pod, where they are using the default pods, so I figured we could add it here as well. I forgot to delete it from `NeuronServer` though

Reply:
I think we should take the `Auth0Pod` out of here b/c Rhizome is meant to be as small/generic as possible and is technically independent from the OpenLattice stack

Reply:
We could actually add another class that extends `RhizomeApplicationServer` that includes `Auth0Pod`, `JdbcPod` and all of the other pods that are always present across `Datastore`  and `Conductor`

Reply:
Okay, that seems like a better idea!